assets/queries/terraform/alicloud/cs_kubernetes_node_pool_auto_repair_disabled/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "id": "81ce9394-013d-4731-8fcc-9d229b474073",
+  "queryName": "CS Kubernetes Node Pool Auto Repair Disabled",
+  "severity": "MEDIUM",
+  "category": "Insecure Configurations",
+  "descriptionText": "Verifies if Alicloud Container Service Node Pool Auto Repair is Enabled",
```

Comment:
Please, improve the `descriptionText` in this query and in the following ones:

- `kics\assets\queries\ansible\gcp\google_container_node_pool_auto_repair_disabled\metadata.json`
- `kics\assets\queries\terraform\gcp\google_container_node_pool_auto_repair_disabled\metadata.json`